_includes/manuals/nightly/1.2_release_notes.md

```diff
@@ -8,6 +8,17 @@ This section will be updated prior to the next release.
 
 ### Deprecations
 
+#### Puppet host methods
+
+Methods accessing puppet ENC information on Host are now deprecated and new macros that are meant to be used in templates were introduced.
+Please stop using the old methods and use the provided macros instead.
```

Comment:
To me this doesn't make sense. IMHO this isn't ENC information but it's really macros to bootstrap Puppet in templates. Also, please always write Puppet with a capital P.
```suggestion
Methods on the Host object used to configure Puppet have been deprecated in favor of dedicated macros. The old methods will issue a deprecation warning and be removed in a future release.
```